Makefile

```diff
@@ -48,6 +52,12 @@ uninstall:
 
 installables: clean bootstrap
 	$(BUILD_TOOL) $(XCODEFLAGS) install
+	rm -rf "$(SWIFTLINTFRAMEWORK_BUNDLE)/Versions/Current/Frameworks/SourceKittenFramework.framework/Versions/Current/Frameworks"
+	xcrun swift-stdlib-tool --copy \
+		--scan-executable "$(SWIFTLINT_EXECUTABLE)" \
+		--scan-folder "$(SWIFTLINTFRAMEWORK_BUNDLE)" \
+		--platform macosx --destination "$(SWIFTLINTFRAMEWORK_BUNDLE)/Versions/Current/Frameworks" \
+		--strip-bitcode
```

Comment:
I copied options that Xcode doing when `EMBEDDED_CONTENT_CONTAINS_SWIFT=YES`.
It seems Xcode uses it regardless of target's platform.
In current behavior, `swift-stdlib-tool` with `--strip-bitcode` just copies Swift libraries without modification.
It might not be necessary, but I couldn't find any reason to drop it that Xcode using.